src/tools/pmreorder/binaryoutputhandler.py

```diff
@@ -168,6 +168,13 @@ def __str__(self):
             hex(self._map_max - self._map_base)
         )
 
+    def not_in(self, files):
+        for f in files:
+            if self._map_max - self._map_base == f._map_max - f._map_base and self._map_base == f._map_base:
```

Comment:
Do you need to compare addresses? Would the same file, but mapped twice with a different address, be correct?